starter_code/views/movies.hbs

```diff
@@ -0,0 +1,10 @@
+
+<ul class="movies">
+    {{#each movies}}
+    <li class="movie-card">
+        <img src="{{image}}" alt="" class="img">
+        <p class="title">{{title}}</p>
+        <a href="movies/{{this._id}}" class="link">See more</a>
```

Comment:
```suggestion
        <a href="movies/{{_id}}" class="link">See more</a>
```

Do jeito que está, funciona perfeitamente. Mas, como vocês não utilizaram o `this` na imagem, nem no titulo, não é necessário utilizar  o `this`.